14/umbraco-cms/fundamentals/code/subscribing-to-notifications.md

```diff
@@ -143,22 +180,26 @@ public class LogWhenPublishedHandler : INotificationHandler<ContentPublishedNoti
         {
             _logger.LogInformation("{ContentName} was published", publishedItem.Name);
         }
-    }
 }
 ```
 
-Now we're ready to spin up our site and give it a try. Go to the Umbraco backoffice and publish a piece of content. Switch to the Settings section and find the Log Viewer in the Advanced menu:
+{% endcode %}
+
+4. Go to the Umbraco backoffice and publish a piece of content.
+5. Check the log messages in the **Log Viewer** under the **Settings** section.
 
-![Log Viewer](images/log-viewer-v14.png)
+    ![Log Viewer](images/log-viewer-v14.png)
 
-Search 'All Logs', and if all is wired up correctly you should discover your custom publish log message entries:
+6. Search 'All Logs'. If everything is set up correctly, you should see your custom log messages.
 
-![Messages in Log](images/log-messages-v14.png)
+    ![Messages in Log](images/log-messages-v14.png)
```

Comment:
Just a thought:

Steps 4-6 are all about checking that the logs work, right?
Should they have their own sub heading, instead of being part of "Registering the Handler"?